striim-transform/README.md

```diff
@@ -0,0 +1,79 @@
+# Striim Compatible Transformer
+
+[Striim](https://www.striim.com) is a distributed data integration and intelligence platform that can be used to design, deploy, and run data movement and data streaming pipelines.
+
+This repository contains simple steps to set up a YugabyteDB CDC pipeline with `StriimCompatible` transfomer that transforms the messages being sent to kafka in a format compatible with Striim's [WAEvent](https://www.striim.com/docs/en/postgresql-cdc.html#UUID-bb4c7685-acc1-6f6a-f4a2-afbfecf4febf).
+
+## Setup
+
+To use `StriimCompatible` transform with the connectors the following properties needs to be added to the connector configuration.
+
+```json
+"transforms": "unwrap",
+"transforms.unwrap.type": "io.debezium.connector.yugabytedb.transforms.StriimCompatible",
+```
+
+**NOTE**: For this transform to work correctly you need to enable before image while creating the cdc stream. You can do it using the following command.
```

Comment:
I am guessing this is because of the fact that the transformer needs the full image of the record?

Reply:
Yes, that's correct.